### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by zootos**
Deployment of the initial internal pages including the members list, see #212 for details.

### CODE REVIEW

File: internal/templates/internal/add_member.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                    <label>{{ form.user.label }}</label>
+                    {{ form.user }}
+                </div>
+                <div class="ui field">
+                    <label>{{ form.date_joined.label }}</label>
+                    {{ form.date_joined }}
+                </div>
+            </div>
+            <div class="ui field">
+                <label>{{ form.committeess.label }}</label>
```
Comment (hermabe) [inline internal/templates/internal/add_member.html]: An `s` too much. The label is not shown.

Comment (zootos) [reply internal/templates/internal/add_member.html]: Do not know how that slipped through. Have fixed it nonetheless.

File: locale/nb/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -217,6 +225,267 @@ msgctxt "abort action"
...
+msgid "No"
+msgstr "Nei"
+
+#: internal/templates/internal/members.html:149
+msgid "Basic information"
+msgstr "Grunnleggende informasjon"
+
+#: internal/templates/internal/members.html:166
+#: make_queue/templates/make_queue/course/course_panel.html:92
+msgid "Card number"
```
[the remark this answers is not in the document]
Comment (zootos) [reply locale/nb/LC_MESSAGES/django.po]: Yeah, that sound right

File: locale/nb/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -217,6 +225,267 @@ msgctxt "abort action"
...
+msgid "Set member as quit"
+msgstr "Set medlem som sluttet"
+
+#: internal/templates/internal/members.html:230
+msgid "Set member as not quit"
+msgstr "Set medlem som ikke sluttet"
+
+#: internal/templates/internal/members.html:231
+msgid "Set member as retired"
+msgstr "Set medlem som pensjonert"
```
[the remark this answers is not in the document]
Comment (zootos) [reply locale/nb/LC_MESSAGES/django.po]: Yeah, should also be "Sett" not  "Set"

File: locale/nb/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -217,6 +225,267 @@ msgctxt "abort action"
...
+#: internal/templates/internal/members.html:37
+#: internal/templates/internal/members.html:38
+#: internal/templatetags/member.py:20
+msgid "Inactive"
+msgstr "Inaktiv"
+
+#: internal/templates/internal/members.html:40
+#: internal/templates/internal/members.html:41
+#: internal/templatetags/member.py:18
+msgid "Pang"
```
[the remark this answers is not in the document]
Comment (zootos) [reply locale/nb/LC_MESSAGES/django.po]: That would probably make sense for consistency

**Pull request merged**